src/builders/linux_vm/kernel.rs

```diff
@@ -125,6 +125,32 @@ impl Kernel {
     fn configure() -> Result<()> {
         run_command(["make", "x86_64_defconfig"]).context("Failed to configure kernel")?;
         Self::configure_squashfs()?;
+        Self::configure_cpu_nb(1024)?;
+        Ok(())
+    }
+
+    /// Configure the kernel to allow a specific number of CPU
+    ///
+    /// Assumes that CWD is kernel directory.
+    fn configure_cpu_nb(cpu: usize) -> Result<()> {
```

Comment:
Minor nitpick on naming that we should probably handle separately: _nb_ doesn't immediately communicate short for number. I think _nr_ is more common for that and it can be also observed below from the kernel config naming, so I think we should fix this, but not in this PR. Just commenting on this as I happened to notice it here.

Reply:
That's gonna be re-worked completely soon, I think. I have an opened ticket for reviewing kernel configs.